src/vs/workbench/contrib/terminal/electron-browser/terminalInstance.ts

```diff
@@ -968,8 +987,13 @@ export class TerminalInstance implements ITerminalInstance {
 		}
 	}
 
-	private _onProcessExit(exitCode: number): void {
-		this._logService.debug(`Terminal process exit (id: ${this.id}) with code ${exitCode}`);
+	private _onProcessOrExtensionCallbackExit(exitCode?: number): void {
```

Comment:
Let's switch this back to `_onProcessExit`, even though it's a renderer exiting, there is still a "process" exiting behind the scenes with an exit code.

Reply:
@Tyriar - see my comment above about terminals and processes. I'll revert if you feel strongly

Reply:
@Tyriar - I'd really like to leave this named as it is because it makes it very clear that this can be executed in situations where there is not a process backing it.